### PR TITLE
Standards Maintenance Issue #627

### PR DESCRIPTION
**Pull request by CDR-API-Stream**
Made below changes to `EnergyPlanTariffPeriod` to allow sharing of banded daily supply charges:
- Added `dailySupplyChargeType` - A new ENUM field to indicate if the daily supply charge is single or banded
- Added `bandedDailySupplyCharges` - A new object allowing representation of banded daily supply charges
- changed `dailySupplyCharges` to `dailySupplyCharge`

### CODE REVIEW

File: slate/source/includes/introduction/_fdo.md
Comment (nils-work) [inline slate/source/includes/introduction/_fdo.md:1]: typos -
- endopint
- endopoint
- retier

File: swagger-gen/api/cds_energy.json [hunk 1]
```diff
@@ -1533,10 +1533,45 @@
                             "type": "string",
                             "description": "The end date of the tariff period in a calendar year.  Formatted in mm-dd format"
                         },
-                        "dailySupplyCharges": {
+                        "dailySupplyChargeType": {
+                            "type": "string",
+                            "description": "Specifies if daily supply charge is single or banded. Default value SINGLE is field not provided",
```
Comment (nils-work) [inline swagger-gen/api/cds_energy.json:1538]: Default value **is** `SINGLE` ~~is~~ **if** field not provided

File: slate/source/includes/introduction/_fdo.md
Comment (CDR-API-Stream) [reply slate/source/includes/introduction/_fdo.md:1]: The above typos have been fixed.

File: swagger-gen/api/cds_energy.json (hunk 1 above)
Comment (CDR-API-Stream) [reply swagger-gen/api/cds_energy.json]: The above has been fixed

**Review by markverstege — APPROVED** [review]

**Pull request merged**